<ui version="4.0" >
 <class>deviceInfoWidget</class>
 <widget class="QWidget" name="deviceInfoWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="cardTpyeLabel" >
     <property name="text" >
      <string>Device:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QComboBox" name="devicesComboBox" />
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="4" row="0" column="4" colspan="2" >
    <widget class="QLabel" name="imageLabel" >
     <property name="minimumSize" >
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="driverNameLabel" >
     <property name="text" >
      <string>Driver:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLineEdit" name="driverLineEdit" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer name="horizontalSpacer_5" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="deviceNodeLabel" >
     <property name="text" >
      <string>Device Node:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLineEdit" name="deviceNodeLineEdit" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <spacer name="horizontalSpacer_7" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>92</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="3" row="3" column="0" colspan="2" >
    <widget class="QToolButton" name="detailToolButton" >
     <property name="text" >
      <string>Details >></string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="3" column="5" >
    <widget class="QPushButton" name="donePushButton" >
     <property name="text" >
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3" >
    <spacer name="horizontalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>265</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="6" >
    <widget class="QTextEdit" name="detailTextEdit" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
